go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/aws/amazon-vpc-cni-k8s
 
 go 1.21
+toolchain go1.22.2
```

Comment:
This is strange. Dependabot is adding a toolchain to go.mod file. This kind of interferes with our .go-version file.